add AWS info to icgc-25k-data.md (#624)
icgc-argo/argo-docs: docs/data-access/icgc-25k-data.md

## Accessing ICGC 25K Release Data

> ICGC Release Data and PCAWG data are hosted in two locations: a SFTP sever and an AWS bucket.

Reviewer: ```suggestion
ICGC Release Data and PCAWG data are hosted in two locations: an SFTP server, and an AWS bucket.
```

The SFTP contains public and DACO controlled data, only available to authorized DACO-approved users only. If you previously had DACO access for ICGC 25K data you will continue to have permission to access the SFTP server. If you require DACO approval please see the documentation on [applying for DACO access](./daco/applying.md).

> The AWS bucket contains public and can be accessed without DACO approval.

Reviewer: "public" what? lol
```suggestion
The AWS bucket contains public data and can be accessed without DACO approval.
```

> Both locations contain directories with the following data:

Reviewer 2: Let's rearrange this text to state upfront that there are different sources for open and controlled data:

```md
ICGC Release Data contains both open and controlled access data. [Open access release data](#link-to-open-access-header) is stored on a publicly available Object Storage Bucket. [Controlled access release data](#link-to-controlled-access-header) is hosted on an SFTP server that requires ICGC DACO approval to access.
```

You can connect to this server using any SFTP client of your choice.

> ### AWS details

Reviewer 2: Lets rename this to `Open Access Release Data`

> The AWS bucket is located at: 

Reviewer: strictly speaking, the url is not it's location, but a pointer to it
```suggestion
The AWS bucket is reachable at: 
```

> The AWS bucket is reachable at: 

Reviewer 2: ```md
Open access release data is hosted on a publicly available Object Storage Bucket. This bucket uses the AWS S3 interface and is therefore accessible using any AWS object storage client.

The bucket is reachable at....
```

- **Host**: `https://object.genomeinformatics.org`

No additional authentication is required.

To navigate and explore the data:
```

> aws s3 ls s3://icgc25k-open --endpoint-url https://object.genomeinformatics.org --no-sign-request

Reviewer: naive question: are people expected to know at this point of the instructions that they need an aws client, etc?

Author: We do, as the same is expected for SFTP and filezilla. It was decided at the time, not to endorse.
